mappings/dependencies.json

```diff
@@ -1,1583 +1,1534 @@
-    {
-      "id": "Xamarin.KotlinX.Coroutines.Guava",
-      "dependencies": [
-        "Xamarin.Kotlin.StdLib.Jdk8",
-        "Xamarin.KotlinX.Coroutines.Core.Jvm",
-        "Xamarin.Google.Guava"
-      ]
-    },
-    {
-      "id": "Xamarin.GooglePlayServices.Basement",
-      "dependencies": []
-    },
-    {
-      "id": "Xamarin.GooglePlayServices.Wearable",
-      "dependencies": []
-    },
-    {
-      "id": "Xamarin.AndroidX.WebKit",
-      "dependencies": [
-        "Xamarin.AndroidX.Annotation",
-        "Xamarin.AndroidX.Core"
-      ]
-    },
-    {
-      "id": "Xamarin.AndroidX.Window",
-      "dependencies": [
-        "Xamarin.AndroidX.Annotation",
-        "Xamarin.AndroidX.Collection",
-        "Xamarin.AndroidX.Core",
-        "Xamarin.Kotlin.StdLib",
-        "Xamarin.KotlinX.Coroutines.Android"
-      ]
-    },
-    {
-      "id": "Xamarin.AndroidX.Window.WindowExtensions",
-      "dependencies": [
-        "Xamarin.AndroidX.Annotation"
-      ]
-    },
-    {
-      "id": "Xamarin.AndroidX.Window.WindowJava",
-      "dependencies": [
-        "Xamarin.AndroidX.Core",
-        "Xamarin.AndroidX.Window",
-        "Xamarin.Kotlin.StdLib",
-        "Xamarin.KotlinX.Coroutines.Core"
-      ]
-    },
-    {
-      "id": "Xamarin.AndroidX.Work.Runtime",
-      "dependencies": [
-        "Xamarin.AndroidX.Annotation.Experimental",
-        "Xamarin.AndroidX.Core",
-        "Xamarin.AndroidX.Lifecycle.LiveData",
-        "Xamarin.AndroidX.Lifecycle.Service",
-        "Xamarin.AndroidX.Room.Runtime",
-        "Xamarin.AndroidX.Sqlite.Framework",
-        "Xamarin.AndroidX.Sqlite",
-        "Xamarin.AndroidX.Startup.StartupRuntime",
-        "Xamarin.Google.Guava.ListenableFuture"
-      ]
-    },
-    {
-      "id": "Xamarin.AndroidX.Work.Work.Runtime.Ktx",
-      "dependencies": [
-        "Xamarin.AndroidX.Work.Runtime",
-        "Xamarin.Kotlin.StdLib",
-        "Xamarin.KotlinX.Coroutines.Android"
-      ]
-    },
-    {
-      "id": "Xamarin.Jetbrains.Annotations",
-      "dependencies": []
-    },
-    {
-      "id": "Xamarin.Kotlin.Reflect",
-      "dependencies": [
-        "Xamarin.Kotlin.StdLib"
-      ]
-    },
-    {
-      "id": "Xamarin.Kotlin.StdLib.Jdk7",
-      "dependencies": [
-        "Xamarin.Kotlin.StdLib"
-      ]
-    },
-    {
-      "id": "Xamarin.Kotlin.StdLib.Jdk8",
-      "dependencies": [
-        "Xamarin.Kotlin.StdLib.Jdk7",
-        "Xamarin.Kotlin.StdLib"
-      ]
-    },
-    {
-      "id": "Xamarin.Google.Guava",
-      "dependencies": []
-    },
-    {
-      "id": "Xamarin.KotlinX.Coroutines.Jdk8",
-      "dependencies": [
-        "Xamarin.Kotlin.StdLib.Jdk8",
-        "Xamarin.KotlinX.Coroutines.Core.Jvm"
-      ]
-    },
-    {
-      "id": "Xamarin.KotlinX.Coroutines.Reactive",
-      "dependencies": [
-        "Xamarin.Kotlin.StdLib.Jdk8",
-        "Xamarin.KotlinX.Coroutines.Core.Jvm",
-        "Xamarin.Android.ReactiveStreams"
-      ]
-    }
-  ]
-}
\ No newline at end of file
+[
```

Comment:
This file looks like it was replaced with `config.json`, is that intentional?

Reply:
no.
I need to see what I have done

Reply:
I resolved conflicts manually in github editor. Seems like I pasted `config.json` somehow. Not sure.

Reply:
building fromscratch to regenerate `dependencies.json`

Reply:
fixed